posts/2023-05-30-23.0.0.5.adoc

```diff
@@ -0,0 +1,183 @@
+---
+layout: post
+title: "MicroProfile 6 and Jakarta EE 10 guide updates in Open Liberty 23.0.0.5"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/mbroz2
+author_github: https://github.com/mbroz2
+seo-title: MicroProfile 6 and Jakarta EE 10 guide updates in Open Liberty 23.0.0.5 - OpenLiberty.io
+seo-description: Concurrent with the Open Liberty 23.0.0.5 release, 44 of the openliberty.io guides have been updated to make use of the latest MicroProfile 6 and Jakarta EE 10 specifications.  This release of the runtime includes various bug fixes.
+blog_description: Concurrent with the Open Liberty 23.0.0.5 release, 44 of the openliberty.io guides have been updated to make use of the latest MicroProfile 6 and Jakarta EE 10 specifications.  This release of the runtime includes various bug fixes.
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+open-graph-image-alt: Open Liberty Logo
+---
+= MicroProfile 6 and Jakarta EE 10 guide updates in Open Liberty 23.0.0.5
+Michal Broz <https://github.com/mbroz2>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+Concurrent with the Open Liberty 23.0.0.5 release, 44 of the Open Liberty guides have been updated to make use of the latest MicroProfile 6 and Jakarta EE 10 specifications.  Various bugs have been fixed as part of this release.
+
+In link:{url-about}[Open Liberty] 23.0.0.5:
+
+* <<guids, 44 guides updated to use MicroProfile 6 and Jakarta EE 10>>
+* <<bugs, Notable bug fixes>>
+
+
+Check out link:{url-prefix}/blog/?search=release&search!=beta[previous Open Liberty GA release blog posts].
+
+
+[#run]
+
+== Run your apps using 23.0.0.5
+
+If you're using link:{url-prefix}/guides/maven-intro.html[Maven], here are the coordinates:
+
+[source,xml]
+----
+<dependency>
+    <groupId>io.openliberty</groupId>
+    <artifactId>openliberty-runtime</artifactId>
+    <version>23.0.0.5</version>
+    <type>zip</type>
+</dependency>
+----
+
+Or for link:{url-prefix}/guides/gradle-intro.html[Gradle]:
+
+[source,gradle]
+----
+dependencies {
+    libertyRuntime group: 'io.openliberty', name: 'openliberty-runtime', version: '[23.0.0.5,)'
+}
+----
+
+Or if you're using link:{url-prefix}/docs/latest/container-images.html[container images]:
+
+[source]
+----
+FROM icr.io/appcafe/open-liberty
+----
+
+Or take a look at our link:{url-prefix}/downloads/[Downloads page].
```

Comment:
```suggestion
Or take a look at our link:{url-prefix}/start/#downloads-pkg[Downloads page].
```

Suggestion 1: Link directly to the download table

Reply:
```suggestion
Or take a look at our link:{url-prefix}/start/[Downloads page].
```

Suggestion 2: Use the `/start` URL to avoid HTTP redirect.

Reply:
Resolved with https://github.com/OpenLiberty/blogs/pull/3101/commits/eecc9a8d4956ccd22e3bab67eaa7e58e7910118b